android/src/com/mapswithme/maps/MwmActivity.java

```diff
@@ -2720,4 +2723,31 @@ public void onGlobalLayout()
                               : UiUtils.getStatusBarHeight(getApplicationContext()));
     }
   }
+
+  @Override                                                                                                                                        
```

Comment:
1. Please move this method above all inner classes in this file.
2. Format it please, after you apply the code style scheme.

Reply:
Done